Prog/Equity_Compile_Feature_Data.sas

```diff
@@ -158,6 +158,68 @@ numerator = pop25andoverwcollege_&_years.;
 run;
 */
 
+data percentblack;
```

Comment:
Total population (number of persons) is also needed for the feature. Please add this to the file.

Reply:
added